src/app/garfile/customs/post.controller.js

```diff
@@ -44,7 +44,7 @@ module.exports = (req, res) => {
           if (buttonClicked === 'Save and continue') {
             res.redirect('/garfile/supportingdocuments');
           } else {
-            res.redirect('/home');
+            res.redirect(307, '/garfile/view');
```

Comment:
Updates to the pathing mean that instead of the dashboard, app goes to the summary.

The summary however is invoked as a POST call, so 307 essentially takes the original POST and forwards it to the "garfile/view" which is sufficient, albeit rather hacky.